### PR TITLE
views/shared/authentication etc i18n

### PR DESCRIPTION
**Pull request by yheuhtozr**
## What type of PR is this? (check all applicable)

- [X] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Extracts strings for i18n from authentication related templates. Attached fr locales for testing purposes. Existing translations up to #15055 reflected (hopefully).

## Related Tickets & Documents

#14888

## Added/updated tests?

- [ ] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [X] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Comment by yheuhtozr** [comment]
@citizen428 Is the last error blocking this from further review? In that case, may I ask what command(s) I need to type in local environment to run that one specific test only?

**Comment by cmgorton** [comment]
Hey @yheuhtozr Michael is on vacation for the next few weeks so will likely be slow to respond. I went ahead and asked our other engineers on OSS rotation to review your open PRs. They mentioned several of them have code conflicts and/or failing specs. You will need to rebase your work and resolve them before they can be reviewed. Thanks again for all your help. Let me know if you need more guidance here. 🙏

**Comment by yheuhtozr** [comment]
@cmgorton Thank you for the information. Concretely speaking, I think #15069 and #15098 are technically ready to merge. Then I'll look into this one and #15136, but especially the last one is only updatable (I already changed but cannot push) after the first two merged due to restrictions on my side. If you find any other problems please let me know.

**Comment by yheuhtozr** [comment]
> Let me know if you need more guidance here.

If you don't mind, I'd specifically like to understand the last error exists in this PR. If I read the test result correctly, the error arises from `app/views/shared/authentication/_forem_creator_signup.html.erb`, but; firstly, this template is only called from `devise`, so that I'm not totally sure if I can change anything in the repo (reasonably) to fix it; secondly, how can I see the rendered page by eye? The failing spec doesn't seem to run at pre-commit hook, and I'm not very certain what is happening there. If it is not terribly difficult, an insight will be appreciated as this is the last real roadblock to the completion of view i18n.

**Comment by yheuhtozr** [comment]
@cmgorton Can I ask about ETA for remaining reviews? If they need a few more weeks, I'll temporarily disable mirroring from GitLab to GitHub in order to let my instance keep updated. Note that @citizen428 had already approved most of them so I guess technically you don't need to wait for him.

**Comment by msarit** [comment]
@yheuhtozr Hello! 👋🏾 
Please check if the TravisCI failures are related to your code changes. 🙏🏾

**Comment by yheuhtozr** [comment]
@msarit Could you tell me how to see the failed page (`app/views/shared/authentication/_forem_creator_signup.html.erb`) by eye on an instance, or give me the failed screenshot (it seems generated by the spec, though not sure retrievable)? Sorry that I'm not very familiar with every template in this app.

**Comment by msarit** [comment]
@yheuhtozr Hey hey! As an update, the team is looking into these Cypress failures. I'll get back to you once we have an understanding. Pls be patient and thank you! 🙏🏾

File: app/views/shared/authentication/_forem_creator_signup.html.erb
Comment (Ridhwana) [inline app/views/shared/authentication/_forem_creator_signup.html.erb:55]: @yheuhtozr I looked into the Travis failures and it seem like this change has caused the error.
Screenshot 2021-11-30 at 12 21 23


As you can see from the screenshot above it is not finding the class `js-creator-signup-username-hint`.

When I look at the DOM I see the following, hence as you can see the span is not rendering correctly. 

Screenshot 2021-11-30 at 12 24 01
 

To test this feature on the frontend you can do the following: 

You can either drop your entire database via `rails:db drop`
and enable the feature flag `FeatureFlag.enable(:creator_onboarding)`

or

You can open a rails console via `rails c` and do the following:
```
# Enable the feature flag
FeatureFlag.enable(:creator_onboarding)

# Delete all users 
User.destroy_all

# Set `waiting_on_first_user` to be true
Settings::General.waiting_on_first_user = true

# If you've logged in before set the mascot id to nil
Settings::General.mascot_user_id = nil
```

Now, run your server and navigate to http://localhost:3000/ and you will see the "Forem Creator Setup" form.

**Review by Ridhwana — CHANGES_REQUESTED** [review]
@yheuhtozr thank you for your patience, I've investigated the Travis failures, and it seems as though a change that you've made on the `_forem_creator_signup.html.erb` has caused the issue. I've provided more details in my review :) Let me know if you require further clarity.

File: app/views/shared/authentication/_forem_creator_signup.html.erb
Comment (yheuhtozr) [reply app/views/shared/authentication/_forem_creator_signup.html.erb:55]: Thank you for your time and the detailed guide! It really helped me doing it right. So this is the page showing only once when DB is empty, that's why I didn't know what it was.

Now the spec is green, and as a by-effect, I realized a couple of useless `rubocop:disable Rails/ContentTag` comments (originally meant to cope with [a bug existed in some versions of Rubocop](https://github.com/rubocop/rubocop-rails/issues/556)) still left in some files I contributed. They are supposed to do no harm but you can remove it whenever convenient.

**Review by Ridhwana — APPROVED** [review]
Thanks for implementing the fix @yheuhtozr! Its looking great now :)

**Pull request merged**
